shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -648,6 +662,12 @@ public void onMessage(Object message, BasicMessageChannel.Reply<Object> reply) {
         }
     }
 
+    private void createWindowChangeEvent(String name) {
+        AccessibilityEvent e = obtainAccessibilityEvent(0, AccessibilityEvent.TYPE_WINDOW_STATE_CHANGED);
```

Comment:
Is it ok to always send this announcement for the root node (id: 0)? Or should this actually be the id of the SemanticsNode that represents the new route?

Reply:
According to the android docs, this should always be sent from the root

Reply:
Maybe add a comment about that here? Most other events need to be send for the specific sub-view.

nit: we should probably define a constant ROOT_NODE_ID for this file and use it in the other places in this file as well...

Reply:
Mostly fixed